move GENERATED_CPP to the only place it is used

dagitses opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #76183
* #76179
* #76178
* #75790
* #75800
* #76174
* #76173
* #75699
* #75869
* #75868
* __->__ #75698
* #75964
* #75332
* #75942
* #74849

Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

Commit:
move GENERATED_CPP to the only place it is used

Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]

malfet reviewed (changes requested):
Hmm, if we can't get rid of it yet, we should keep it in build variables

Commit:
Update on "move GENERATED_CPP to the only place it is used"


Note that there is a distinct GENERATED_CPP list in a different bzl
file.

Differential Revision: [D35587411](https://our.internmc.facebook.com/intern/diff/D35587411/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D35587411/)!

[ghstack-poisoned]
